modules/gdscript/language_server/gdscript_text_document.cpp

```diff
@@ -427,7 +427,9 @@ void GDScriptTextDocument::sync_script_content(const String &p_path, const Strin
 	EditorFileSystem::get_singleton()->update_file(path);
 	Ref<GDScript> script = ResourceLoader::load(path);
 	script->load_source_code(path);
-	script->reload(true);
+	if (script.is_valid()) {
```

Comment:
That's weird, if `script->reload(true)` didn't crash then `script` should already be a valid reference. I don't see what would invalidate the reference in `load_source_code()` from a quick look.

CC @Razoric480

Reply:
It doesn't. It triggers `ERR_FAIL_COND_V`. So proper handling would be more akin to:
```cpp
	Error err = script->load_source_code(path);
	if (OK == err) {
		script->reload(true);
	}
```

Reply:
Updated commit accordingly.